server/src/main/java/school/journal/controller/PupilAPIController.java

```diff
@@ -110,37 +112,36 @@ public ResponseEntity getPupilInfo(HttpServletRequest request, @PathVariable("pu
         ResponseEntity resultResponse;
         try {
             LOGGER.info("Get Pupil entity Controller method");
-            resultResponse = new ResponseEntity(pupilService.getPupilInfo(pupilId),
-                    OK);
+            resultResponse = new ResponseEntity(pupilService.getPupilInfo(pupilId), OK);
         } catch (ServiceException exc) {
             LOGGER.error(exc);
-            resultResponse = new ResponseEntity(new ErrorObject("Error in pupil getting"), BAD_REQUEST);
+            resultResponse = new ResponseEntity(new ErrorObject(classSubject, "Get one", exc), BAD_REQUEST);
         } catch (Exception exc) {
             LOGGER.error(exc);
             resultResponse = new ResponseEntity(CRITICAL_ERROR, INTERNAL_SERVER_ERROR);
         }
         return resultResponse;
     }
 
-    @RequestMapping(value = "/{pupilId}", method = GET,params = "classId")
+    @RequestMapping(method = GET, params = "classId")
```

Comment:
if it will be as get parameter, change it to `@RequestMapping(value = "/{pupilId}", method = GET)`